panther_lights/src/lights_driver_node.cpp

```diff
@@ -0,0 +1,149 @@
+#include <panther_lights/lights_driver_node.hpp>
+
+#include <filesystem>
+#include <memory>
+
+#include <gpiod.hpp>
+#include <ros/ros.h>
+
+#include <image_transport/image_transport.h>
+#include <sensor_msgs/image_encodings.h>
+
+#include <panther_msgs/SetLEDBrightness.h>
+
+#include <panther_lights/apa102.hpp>
```

Comment:
```suggestion
#include <filesystem>

#include <sensor_msgs/image_encodings.h>
```
already defined in hpp file

Reply:
You already proved yourself that they should be in both places